pom.xml

```diff
@@ -31,11 +31,18 @@
 		<rulesets.xml.location>${project.basedir}/../config/rules/rulesets.xml</rulesets.xml.location>
 		<checkstyle.xml.location>config/checkstyle/checkstyle.xml</checkstyle.xml.location>
 		<version.wildfly>10.1.0.Final</version.wildfly>
+		<java.ee.version>7.0</java.ee.version>
 	</properties>
 
 	<dependencyManagement>
 		<dependencies>
 			<dependency>
+				<groupId>javax</groupId>
+				<artifactId>javaee-api</artifactId>
```

Comment:
I think it is not a good idea to pull the whole javaee-api because we just want to use the javax.ws.rs-api.

So please remove this dependency from the whole project and use the following:
https://mvnrepository.com/artifact/javax.ws.rs/javax.ws.rs-api/2.0.1